portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+  
+  private String name;
+  private String content;
+  private int numLikes;
+  private Date timestamp;
+
+  public Comment(String name, String content) {
+    this.name = name;
+    this.content = content;
+    numLikes = 0;
+    timestamp = new Date();
+  }
+
+  public Comment(String content) {
+    this("anonymous", content);
```

Comment:
Lets add a param tag to "anonymous" here, which would make this line look like:
this(/* name= */ "anonymous", content);

We use these param tags that contain the argument name used in the method declaration (which is "name" in this case) when its not clear from the passed in value ("anonymous") what its referring to. the second argument "content" is self-evident so its not needed for that. 

Another common example for when param tags are good is when multiple arguments have the same type - so consider a class with constructor Point(int x, int y)
and then someone in another class instantiates a Point object:
new Point(1, 3);
instead of this if they did:
new Point(/* x= */ 1, /* y= */ 3); it would be immediately clear to readers which number is the x and which is the y coord and also help the author not accidentally switch the order of the args (which as you can tell in this case, the compiler would be just fine with since they;re both ints)